src/main/java/com/networknt/schema/OneOfValidator.java

```diff
@@ -209,4 +225,17 @@ public OneOfValidator(String schemaPath, JsonNode schemaNode, JsonSchema parentS
         return validationMessages;
     }
 
+    private ValidationMessage getBothSchemasValidErrorMsg(String at){

```

Comment:
What if we have 3 or more valid schemas ?

Reply:
Thank you @guillomep. I did not name the method properly. Thanks for catching it. I have changed the method name properly.